launcher/sdw_updater_gui/strings.py

```diff
@@ -1,56 +1,41 @@
-# checking for updates
-description_status_checking_updates = (
-    "<h3> Checking for updates... </h3> <p>This should take around 3 to 5 minutes.</p>"
+headline_introduction = "Preflight security updates"
+description_introduction = (
+    "<p>To keep your Workstation safe, daily software updates are required.</p> "
+    "<p>This typically takes between 5 and 30 minutes. You cannot use the SecureDrop "
+    "Client or any of is VMs while the updater is running.</p>"
+    "<p><span style='color:#E62354;'><b>Interrupting software updates may break "
+    "the Workstation.</b></span> Please cancel and return later if you are pressed "
```

Comment:
"Please cancel and shut down the Workstation"?

Reply:
How do you feel the updated language obfuscates or fails to speak to that?

Reply:
I think it will depend on if the user is using non-securedrop VMs on their workstation. At the risk of making this message overly long/complicated, I retract my comment.